.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -37,6 +37,16 @@ assignees: ''
 2. ...
 3. ...
 
+<!--
+     Please tell us on which platform(s) the problem occurs (Android / iOS / Web / Desktop)
```

Comment:
rather than Desktop we should say macOS / Windows or something like that.

Reply:
Yeah was thinking on that too but thought they might confude this with the host platform

Reply:
We should address that by making the question clearer though, not by having the question ask them for the wrong information. If a bug comes to me that happens on "desktop" that doesn't really give me useful information.